var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -241,6 +243,16 @@ def cmake_args(self):
                 '-DDEAL_II_WITH_OPENCASCADE=OFF'
             ])
 
+        # 64 bit indices
+        if '+64bit' in spec:
+            options.extend([
+                '-DDEAL_II_WITH_64BIT_INDICES=ON'
```

Comment:
Can I advocate for a new idiom here?  You know you can do this, right?

```python
'-DDEAL_II_WITH_64BIT_INDICES=%s' % ('+64bit' in spec)
```

That `str(<some bool>)` resolves to `True` or `False`, and `CMake` understands any case variation of `TRUE`/`FALSE`, [among other things](https://cmake.org/Wiki/CMake/Language_Syntax#CMake_supports_boolean_variables.).  Even things ending in `-NOTFOUND` are false to `CMake`, though that was always a bit too much like a [daily WTF](http://thedailywtf.com/articles/What_Is_Truth_0x3f_) for my taste.